pages/common/finger.md

```diff
@@ -0,0 +1,23 @@
+# finger
+
+> User information lookup program.
+
+- Display the user's login name, real name, terminal name and write status (as a "\*" after the terminal name if write permission is denied), idle time, login time, office location and office phone number:
+
+`finger -s`
+
+- Produce a multi-line format displaying all the information described for the option ```-s``` as well as the user's home directory, home phone number, login shell, mail status, and the contents of the files ".plan", ".project", ".pgpkey" and ".forward" from the user's home directory:
```

Comment:
This is copied verbatim from the man page. Please shorten it to a simpler version, and if possible use a word starting with `l`.

Reply:
Shortened it up, but no word came to mind that starts with an `l`.